### ISSUE TITLE
No remotes match the current setting. But logs show it found remote

### ISSUE THREAD

**Issue by rafaelsmgomes** [microsoft/vscode-pull-request-github#4640]
I can't use the extension because it does not show anything on the sidebar. It keeps saying that no remotes match the current setting, but when I check the logs it shows that it found the Git repository.

![image](https://user-images.githubusercontent.com/52176516/225950370-702b8feb-8ebc-4751-959d-87baa9668842.png)

Here are the logs again:
```bash
2023-03-17 10:20:52.312 [info] Looking for git repository
2023-03-17 10:20:52.312 [info] Found 0 repositories during activation
2023-03-17 10:20:52.312 [info] Git repository found, initializing review manager and pr tree view.
2023-03-17 10:20:52.314 [info] Registering git provider
2023-03-17 10:20:52.314 [info] Review> Validate state in progress
2023-03-17 10:20:52.314 [info] Review> Validating state...
2023-03-17 10:20:52.333 [info] Found GitHub remote for folder /Users/rafaelgomes/portfolio/sites
2023-03-17 10:20:52.384 [info] Repo state for file:///Users/rafaelgomes/portfolio/sites changed.
2023-03-17 10:20:52.384 [info] Repo file:///Users/rafaelgomes/portfolio/sites has already been setup.
2023-03-17 10:20:52.385 [info] Review> no matching pull request metadata found for current branch main
2023-03-17 10:20:52.385 [info] Review> No matching pull request metadata found on GitHub for current branch main
```

- Extension version: 0.60.0
- VSCode Version: 1.76
- OS: macOS Monterrey 12.5.1

**Comment by alexr00** [comment]
Can you share the output of `git remote -v`?

**Comment by cino** [comment]
I am actually experiencing the same issue and not able to figure out why. It looks like the remote is found but still no luck. 

![image](https://user-images.githubusercontent.com/2735602/235920303-374aaf56-12b4-453a-a6a0-d144e9221864.png)

Extension version: v0.62.0
VSCode Version: Version: 1.77.3
OS: MacOS Ventura 13.2.1  / Darwin arm64 22.3.0

The remotes are as the default is configured with origin and opstream.

![image](https://user-images.githubusercontent.com/2735602/235920391-a13b23f4-cf17-40dc-8b35-5b3e2615d0d6.png)

The actual remote is from origin (also default)

![image](https://user-images.githubusercontent.com/2735602/235920223-3311f65e-aa97-4ba8-886a-15e55f1e9b38.png)


The logs in output: 

> 2023-05-03 14:40:04.697 [info] Looking for git repository
2023-05-03 14:40:04.697 [info] Found 0 repositories during activation
2023-05-03 14:40:04.697 [info] Git repository found, initializing review manager and pr tree view.
2023-05-03 14:40:04.699 [info] Registering git provider
2023-05-03 14:40:04.699 [info] Review> Validate state in progress
2023-05-03 14:40:04.699 [info] Review> Validating state...
2023-05-03 14:40:04.708 [info] Found GitHub remote for folder /Users/ricardocino/Dev/Private/vscode-pr-test
2023-05-03 14:40:04.773 [info] Repo state for file:///Users/ricardocino/Dev/Private/vscode-pr-test changed.
2023-05-03 14:40:04.773 [info] Repo file:///Users/ricardocino/Dev/Private/vscode-pr-test has already been setup.
2023-05-03 14:40:04.773 [info] Review> no matching pull request metadata found for current branch main
2023-05-03 14:40:04.773 [info] Review> No matching pull request metadata found on GitHub for current branch main
2023-05-03 14:40:14.568 [info] Review> Validate state in progress
2023-05-03 14:40:14.568 [info] Review> Validating state...
2023-05-03 14:40:14.578 [info] Found GitHub remote for folder /Users/ricardocino/Dev/Private/vscode-pr-test
2023-05-03 14:40:14.595 [info] Review> no matching pull request metadata found for current branch update-readme
2023-05-03 14:40:14.595 [info] Review> No matching pull request metadata found on GitHub for current branch update-readme
2023-05-03 14:40:26.300 [info] Review> Validate state in progress
2023-05-03 14:40:26.300 [info] Review> Validating state...
2023-05-03 14:40:26.337 [info] Found GitHub remote for folder /Users/ricardocino/Dev/Private/vscode-pr-test
2023-05-03 14:40:26.360 [info] Review> no matching pull request metadata found for current branch update-readme
2023-05-03 14:40:26.360 [info] Review> No matching pull request metadata found on GitHub for current branch update-readme
2023-05-03 14:40:29.794 [info] Review> Validate state in progress
2023-05-03 14:40:29.794 [info] Review> Validating state...
2023-05-03 14:40:29.806 [info] Found GitHub remote for folder /Users/ricardocino/Dev/Private/vscode-pr-test
2023-05-03 14:40:29.825 [info] Review> no matching pull request metadata found for current branch update-readme
2023-05-03 14:40:29.825 [info] Review> No matching pull request metadata found on GitHub for current branch update-readme


Not sure what I can do to make it work at this point, if you need more information i'm happy to provide.

**Comment by alexr00** [comment]
Thanks for the additional info @cino! It does look like it's finding your remotes correctly. Can you share the output from the developer console (command Developer: Toggle Developer Tools and then click on the "Console" tab in the dev tools)?

**Comment by cino** [comment]
@alexr00 Sure, when I enable the developer console and reload the VSCode window via the Reload Window command this is all I get in the developer console: 

![image](https://user-images.githubusercontent.com/2735602/235940402-2379ad19-0a03-4376-8754-306a148f394d.png)

Also no additional entries when trying to refresh the pull request / issue list

**Comment by cino** [comment]
@alexr00 Yes, the account that is logged in is connected to an enterprise organisation but not on a self-hosted environment, all on github.com. I've tested this both with and without authorising to the enterprise (saml) organisation. 

The repository I'm using as a test is not in the enterprise though.

**Comment by alexr00** [comment]
@cino just to confirm then: you use the same GitHub account to access both your organization repos and your private repos?

**Comment by cino** [comment]
@alexr00 That's correct!

**Comment by rafaelsmgomes** [comment]
Any news on this?

**Comment by mlowitz** [comment]
hey i found the issue if you created your repo differently the remote name isn't origin its the repo name .  add the repo name to remotes

**Comment by juandeGLS** [comment]
@mlowitz thanks!!!
